contracts/proxy/ERC1967/ERC1967Upgrade.sol

```diff
@@ -114,7 +132,8 @@ abstract contract ERC1967Upgrade is IERC1967 {
      * This is the keccak-256 hash of "eip1967.proxy.admin" subtracted by 1, and is
      * validated in the constructor.
      */
-    bytes32 internal constant _ADMIN_SLOT = 0xb53127684a568b3173ae13b9f8a6016e243e63b6e8ee1178d6a717850b5d6103;
+    // solhint-disable-next-line private-vars-leading-underscore
```

Comment:
I want to raise a question about this. It's not the first time I found cases where this warning shows and I actually think the constants should not have a leading underscore, but it's becoming quite common.

Can we take a look a the rule configuration? I see in the docs there's [a `strict`](https://github.com/protofire/solhint/blob/master/docs/rules/naming/private-vars-leading-underscore.md) flag that might help, but only apply to [non-state vars](https://github.com/protofire/solhint/blob/master/lib/rules/naming/private-vars-leading-underscore.js#L82), which I'm not sure if constants fall into that category.

Non blocking in case this convo gets bigger.

Reply:
100% we don't want this rule, we can't have these comments everywhere. Solhint is not working well for us...

Reply:
Lets create an issue for that, and resolve in another PR

Reply:
https://github.com/OpenZeppelin/openzeppelin-contracts/issues/4355